ag_data/tests/test_utilities.py

```diff
@@ -39,7 +39,7 @@ def test_createOrResetBuiltInSensorTypeAtPresetIndex_reset(self):
 
             sensorType = AGSensorType.objects.get(pk=reference["agSensorTypeID"])
 
-            sensorType.name = reference["agSensorTypeName"] + " "
+            sensorType.name = reference["agSensorTypeName"].join(" ")
```

Comment:
It might not be a critical issue but i guess `reference["agSensorTypeName"] + " "` and `reference["agSensorTypeName"].join(" ")` have different behavior. Can you confirm that it's okay?

Reply:
You are correct. I'm changing it back now.